packages/components/src/Filter/FilterBar/subcomponents/FilterBarMultiSelect/FilterBarMultiSelect.tsx

```diff
@@ -94,12 +93,13 @@ export const FilterBarMultiSelect = ({
                 items
               )
             : [],
-          label: filterState.name,
+          label: label || filterState.name,
```

Comment:
If we are to allow overriding, we'd want it to flow onto all the instances of `filterState.name`, so you'll want to ensure L77 ~~and L102~~ also follow the pattern (you can make a variable).

Any usages which want the labels to differ (such as your use case) will need to be a customisation of the `trigger` prop by the consumer instead of being the default :)

Reply:
https://github.com/cultureamp/kaizen-design-system/pull/4604#discussion_r1552733743